jaas/src/main/java/com/griddynamics/jagger/jaas/storage/model/JobEntity.java

```diff
@@ -29,8 +31,8 @@
     private Long jobStartTimeoutInSeconds;
 
     @JsonIgnore
-    @OneToOne(mappedBy = "job", cascade = CascadeType.REMOVE)
-    private JobExecutionEntity jobExecutionEntity;
+    @OneToMany(mappedBy = "job", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<JobExecutionEntity> jobExecutionEntities;
```

Comment:
It's needed for hibernate to register relation. I'm not sure, if it possible to make this relation without this field in JobEntity. I can try to eliminate it.